sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ValueStopwatch.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.ServiceBus.Primitives
+{
+    using System;
+    using System.Diagnostics;
```

Comment:
nit:  We should elevate these before the namespace to match the convention used elsewhere.

Reply:
Fixed